### PR TITLE
adds poster image option to video widget

### PR DESCRIPTION
**Pull request by digitaldan**
Signed-off-by: Dan Cunningham <dan@digitaldan.com>

### CODE REVIEW

File: bundles/org.openhab.ui/web/src/components/widgets/system/oh-video.vue [hunk 1]
```diff
@@ -6,7 +6,8 @@
       :stunServer="config.stunServer"
       :candidatesTimeout="config.candidatesTimeout"
       :startManually="config.startManually"
-      :hideControls="config.hideControls" />
+      :hideControls="config.hideControls"
+      :posterURL="config.posterURL" />
```
Comment (ghys) [inline bundles/org.openhab.ui/web/src/components/widgets/system/oh-video.vue:10]: Wouldn't you need to add this to the `<oh-video-videojs>` alternative below too?

**Review by ghys — COMMENTED** [review]
Thanks for the change (and including the doc update!). One small question:

File: bundles/org.openhab.ui/web/src/components/widgets/system/oh-video.vue (hunk 1 above)
Comment (digitaldan) [reply bundles/org.openhab.ui/web/src/components/widgets/system/oh-video.vue:10]: 🤦   Good catch, i somehow lost that in a refactor.  Fixed.

**Review by ghys — APPROVED** [review]
Thanks!

**Pull request merged**